Doberman/Node.py

```diff
@@ -0,0 +1,315 @@
+import Doberman
+
```

Comment:
import requests and itertools

Reply:
And also remove them from Pipeline.py. A relic of when the files were combined.